nix/nixpkgs.nix

```diff
@@ -1,8 +1,19 @@
-# Pinned nixpkgs version.
+# Pinned nixpkgs version with Rust overlay.
 
 let
-  stableSrc =
+  rust-overlay-src =
     let
+      # When you change this, also change the sha256 hash!
+      rev = "e2ceeaa7f9334c5d732323b6fec363229da4f382";
+    in
+    builtins.fetchTarball {
+      url = "https://github.com/oxalica/rust-overlay/archive/${rev}.tar.gz";
+      sha256 = "sha256:1rs8k4fs46vkw21ahc1zqa57hlaibcgqkqqm2g5x4pkkgh762nk0";
+    };
+  rust-overlay = import rust-overlay-src;
```

Comment:
I think you might want to look into flakes or niv.

Reply:
yes, I switched to niv. The old approach was way to unhandy